contrib/sway/template.py

```diff
@@ -0,0 +1,44 @@
+pkgname = "sway"
+pkgver = "1.8.1"
+pkgrel = 0
+build_style = "meson"
+configure_args = ["-Dwerror=false"]
+hostmakedepends = [
+    "meson",
+    "pkgconf",
+    "ninja",
```

Comment:
ninja doesn't need explicit hostdep (it's always installed with meson); ditto everywhere else